src/modules/json.wren

```diff
@@ -0,0 +1,245 @@
+import "io" for FileSystem
+
+class JsonOptions {
+    static NIL { 0 }
+    static ESCAPE_SLASHES { 1 }
+    static ABORT_ON_ERROR { 2 }
+
+    static shouldAbort(options) {
+        return (options == JsonOptions.ABORT_ON_ERROR || options == (JsonOptions.ABORT_ON_ERROR | JsonOptions.ESCAPE_SLASHES))
```

Comment:
This can be simplified to: `return options & JsonOptions.ABORT_ON_ERROR != 0`, which will be much more scalable, if extra options are added in the future.

Reply:
done